### DISCUSSION TITLE
How to contribute fixes to linuxbrew?

### DISCUSSION THREAD

**Discussion by chandlerc** [Homebrew/discussions#2060] [Linux] [answered]
Hello,

I've submitted a few PRs to Linuxbrew in the past by sending direct PRs. These were Linux-specific patches. I have another one but can no longer open PRs there. I guess I can send the patch to the `homebrew-core` repository instead?

Mostly, I wanted to flag that the contributing guide seems out of date:
https://github.com/Homebrew/linuxbrew-core/blob/HEAD/CONTRIBUTING.md

There it says to send PRs to Linuxbrew if unsure, but that seems clearly no longer the case?

Also, I don't seem to be able to file issues against that repository either.

Anyways, mostly wanting to get the contributing docs updated.

**Comment by carlocab** [comment]
Oops, yes, those docs are out of date. Please open any issues/PRs at Homebrew/core instead.

**Comment by carlocab** [comment, marked as answer]
I've opened Homebrew/linuxbrew-core#24382 to update `CONTRIBUTING.md`. Thanks for the reminder, and apologies for the confusion!
